### ISSUE TITLE
Update the Module store page 

### ISSUE THREAD

**Issue by SlnPons** [massalabs/thyra#688]
**Context**
"Plugins store" page is renamed "Modules store" page. It's less techy and simpler for end-users. 
The objective of this issue is to re-design et update the module page so it's mainnet ready.

**User flow**

**Designs**
Page deisgn: https://www.figma.com/file/LSaOibVDFb1reF6FePVn5l/Massastation?type=design&node-id=1026-11415&t=rtJbo6TfzywZSEcI-0

**How to**

- [ ] Create page with new components (front-end)
- [ ] Connect page to back-end and make appropriate back-end changes
- [ ] Create and implement rules (errors, and others)

**Technical details**
*Give the technical insights so anyone in the team can tackle the tasks - Dev*

**QA testing**
Follow user flow described

**Comment by SlnPons** [comment]
@pivilartisant @Pierregvx looking at the action plan, I think we're missing a point which is: connect back-end and front-end for "My station" section? 
(same for other sections, but let' start with this one first ;) ) 

WDYT? 
I would be keen to put in done the page step by step rather than doing all FE then all BE etc... but does it make sense from a technical perspective?

**Comment by SlnPons** [comment]
Just added the steps in the description

**Comment by pivilartisant** [comment]
@SlnPons agreed, our goal is to advance in increments when we have no blockers, otherwise if we are block or have to many external dependancies on a task to advance another one !

**Comment by Pierregvx** [comment]
can we close this @pivilartisant ?

**Comment by SlnPons** [comment]
After testing on the main branch pulled on June 27th at 3pm, I can see this problem: 

- [ ] Account drop down is at 2 different places. Is that normal. It looks weird ....
- [ ] The upload website slider: you can't have a website with 2 characters, why?
- [ ] It is taking a loooooooon time to display the page. Why's that? 
Can we explain these long minutes and identify how to improve this (probably later but this is not really satisfying :/) 

(niceee faviconnn)
